docs/concepts/automatic-data-locality.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Automatic data locality"
+linkTitle: "Automatic data locality"
+weight: 1
+---
+
+## Ondat volumes move with their pods
+
+Having an Ondat volume instance on the same node as the pod using it can bring some benifits to your application. Unfortunatly pods can move around over time leaving the Volume behind.
+
+Automatic-Data-Locality mitigates that, making the primary Ondat volume instance follow the pod as it moves between cluster nodes.
```

Comment:
By volume instance, we mean the primary deployment right? Is it better to say primary deployment? I don't think we use the terminology "primary Ondata volume instance" else where so it could be confusing.

Reply:
I was told we don't use "master" or "deployment" on our public docs although after looking around I see this is not actually true so I will change all my uses of "instance" into "deployment"